recipes/frugally-deep/bld.bat

```diff
@@ -0,0 +1,19 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "NMake Makefiles" ^
```

Comment:
If you depend on `Ninja` on Windows, probably you may want to use the `Ninja` generator.

Reply:
Sure, for long builds `Ninja` is tipically faster, but in this case it does not play a big role.

Reply:
In https://github.com/conda-forge/staged-recipes/pull/20583/commits/f69dc80b580fa9b98e145848689ac180e01d90df I updated the recipe to avoid asking for `Ninja`.